go.mod

```diff
@@ -34,7 +34,8 @@ require (
 	github.com/otiai10/copy v1.1.1
 	github.com/pborman/uuid v0.0.0-20180906182336-adf5a7427709
 	github.com/robfig/cron v1.2.0
-	github.com/sirupsen/logrus v1.8.1
+	github.com/sirupsen/logrus v1.9.0
+	github.com/startreedata/pinot-client-go v0.0.0-20230303070132-3b84c28a9e95
```

Comment:
nit: maybe a comment on why this particular version is needed

Reply:
Sure, their last release (0.1.0) was from 2022, so we used master here.